### ISSUE TITLE
”Search region” is not present in the page

### ISSUE THREAD

**Issue by maggi1129** [microsoft/BotFramework-WebChat#2237]
## Version

OS : WIN 10
Browser : Chrome
Tools:JAWS
Webchat Version : "4.3.1-master.98c662f"

## Describe the bug

Screen Reader users will get impacted if “Search message” region is not present in the page

## To Reproduce

Repro Steps:
Step 1: Enable Jaws  and open the above URL in latest Chrome browser .
Step 2: Navigate to the page using cursor mode(down arrow)
Step 3: Use "Insert + Ctr + R" to open region window.
Step 4: Observe and verify whether “search message”  region is present in the page using “R” key stroke

## Expected behavior

"Search region” should be present in the page

## Additional context

[Bug]

**Comment by corinagum** [comment]
@maggi1129 thank you for filing this issue! I am waiting for a response from the accessibility team to see what browser(s) we should support JAWS in. I will update this issue when I hear back from them. :)

**Comment by corinagum** [comment]
@maggi1129, thanks for your patience. I confirmed with the accessibility team that we want to support JAWS and Chrome. I will inform the team about this issue and get back to this once I have done some more research and have more information available.

**Comment by cwhitten** [comment]
@maggi1129 we suggest using the following supported screen reader technology. We do not officially support JAWS.

**Narrator** on MS Edge (non-chromium)
**NVDA** on Chrome, Firefox, MS Chromium Edge
**VoiceOver** on Safari
